R/split_moment_matching.R

```diff
@@ -0,0 +1,128 @@
+#' Split moment matching for efficient approximate leave-one-out cross-validation (LOO)
+#'
+#' A function that computes the split moment matching importance sampling loo.
+#' Takes in the moment matching total transformation, transforms only half
+#' of the draws, and computes a single elpd using multiple importance sampling.
+#'
+#'
+#' @export split_mmloo
```

Comment:
do we need to export split_mmloo at this point or shall we keep it internal and only make it callable via mmloo?

Reply:
That is probably better. I will do that unless someone has a different opinion